modules/ROOT/pages/_partials/upgrade-tool.adoc

```diff
@@ -0,0 +1,82 @@
+//This content is reused in:
+//mule-upgrade-tool.adoc
+
+// Before You Begin
+// tag::BeforeYouBegin[]
+
+* Ensure that you have xref:release-notes::mule-upgrade-tool/mule-upgrade-tool.adoc[the latest available version of Mule upgrade tool] so that the tool runs with the latest fixes and security enhancements.
+* A currently operational Mule instance running version 4.1.1 or later, in _stopped_ status to prepare for the upgrade.
++
+When updating your current Mule instance from a version between 4.1.1 and 4.1.4, use any patch update released after January 20, 2022. If the running version is not one of the previously mentioned, an error message appears stating that the Mule version cannot be upgraded unless an upgrade is made.
+* The Mule upgrade tool inside your current Mule instance's `$MULE_HOME/tools` directory, that is, your old or existing Mule instance. 
++
+To download the latest version of the tool (recommended), log in to https://help.mulesoft.com/s/[Help Center^]. Under the Support menu, select *Downloads > Mule Runtime*.
++
+An example of the file name is `mule-runtime-upgrade-tool-1.1.0.zip`.
+* The latest Mule runtime engine distribution, downloaded and unpacked in your system.
```

Comment:
So we are telling them to upgrade to the latest version of Mule and not to some earlier version of Mule? I'm not sure how to reconcile that with what's above:

"When updating your current Mule instance from a version between 4.1.1 and 4.1.4, use any patch update released after January 20, 2022."